tests/CMakeLists.txt

```diff
@@ -94,8 +94,8 @@ endif()
 
 if(AVIF_ENABLE_GTEST)
     if(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
-        add_avif_gtest_with_data(avif16bittest)
-        add_avif_gtest(avifsampletransformtest)
+        add_avif_internal_gtest_with_data(avif16bittest)
+        add_avif_internal_gtest(avifsampletransformtest)
```

Comment:
Yannis: This means none of the CI workflows test the experimental features in a shared library build. Could you suggest a suitable CI workflow where we can do this? Or do we need to add a new CI workflow?

Reply:
Since it is in `ci-unix-static.yml` I guess it makes sense to add it to `ci-unix-shared-local.yml`. I expect the other experimental flags to be untested in a shared library build too.

Reply:
Thanks for the suggestion. I will do that in https://github.com/AOMediaCodec/libavif/pull/2307 so that I can first show that these tests have linker errors in shared library builds when the experimental features are enabled.